eva/third_party/huggingface/create.py

```diff
@@ -46,6 +46,7 @@
     "image-to-text": HFInputTypes.IMAGE,
     "object-detection": HFInputTypes.IMAGE,
     "depth-estimation": HFInputTypes.IMAGE,
+    "ner": HFInputTypes.TEXT,
```

Comment:
nit: Move it along with the other `HFInputTypes.TEXT` types.

Reply:
made this changes and pushed